<ui version="4.0" >
 <class>SvnImportDialogWidget</class>
 <widget class="QWidget" name="SvnImportDialogWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>197</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Source file or directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="srcUrlRequester" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Target repository Url. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="destUrlRequester" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="nonrecurseChk" >
     <property name="text" >
      <string>Non recursive</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="noignoreChk" >
     <property name="text" >
      <string>disregard default and svn:ignore property ignores</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
